beacon_node/lighthouse_network/src/rpc/protocol.rs

```diff
@@ -390,16 +363,18 @@ impl ProtocolId {
                 <OldBlocksByRangeRequestV2 as Encode>::ssz_fixed_len(),
                 <OldBlocksByRangeRequestV2 as Encode>::ssz_fixed_len(),
             ),
-            Protocol::BlocksByRoot => {
-                RpcLimits::new(*BLOCKS_BY_ROOT_REQUEST_MIN, *BLOCKS_BY_ROOT_REQUEST_MAX)
-            }
+            Protocol::BlocksByRoot => RpcLimits::new(
+                fork_context.blocks_by_root_request_min,
+                fork_context.blocks_by_root_request_max,
```

Comment:
can't we just put these in the spec? 

They may not be "spec'd" but we can set default values if they don't exist in the config file and I think the trend is to move these kinds of configs in there anyway.

Reply:
I didn't put them there initially because they're kind of clunky in the spec due to our overriding logic with file configs. We have our spec defaults, which we'll need to derive these from, and then we apply `Config` (file config) on top of the defaults and we aren't sure if our default configs changed so we need to re-derive them. that said, I added it there now and it's not too bad.